File: TrabajoFinalPrueba1/PegasusV1/PegasusV1/Entities/IntegrantesEventos.cs
﻿using System.ComponentModel.DataAnnotations.Schema;

namespace PegasusV1.Entities
{
    [Table("INTEGRANTES_EVENTOS")]
    public class IntegrantesEventos
    {
        public int Id { get; set; }

        [ForeignKey("Id_Evento")]
        public Evento? Evento { get; set; }
        public int? Id_Evento { get; set; }

        [ForeignKey("Id_Usuario")]
        public Usuario? Usuario { get; set; }
        public int? Id_Usuario { get; set; }
        public bool Leido { get; set; }
        public bool Confirmado { get; set; }

    }
}

File: pegasusapp/src/models/integrantesEventos.ts
import { Usuario } from './usuario';
import { Evento } from './evento';

export interface IntegrantesEventos {
  id: number
  evento?: Evento | null
  id_Evento?: number | null
  usuario?: Usuario | null
  id_Usuario?: number | null
  leido?: boolean
  confirmado?: boolean
}
